config/set/code-quality.php

```diff
@@ -2,191 +2,15 @@
-        ExplicitBoolCompareRector::class,
-        CombineIfRector::class,
-        UseIdenticalOverEqualWithSameTypeRector::class,
-        SimplifyBoolIdenticalTrueRector::class,
-        SimplifyRegexPatternRector::class,
-        BooleanNotIdenticalToNotIdenticalRector::class,
-        CallableThisArrayToAnonymousFunctionRector::class,
-        AndAssignsToSeparateLinesRector::class,
-        CompactToVariablesRector::class,
-        CompleteDynamicPropertiesRector::class,
-        IsAWithStringWithThirdArgumentRector::class,
-        StrlenZeroToIdenticalEmptyStringRector::class,
-        ThrowWithPreviousExceptionRector::class,
-        RemoveSoleValueSprintfRector::class,
-        ShortenElseIfRector::class,
-        ExplicitReturnNullRector::class,
-        ArrayMergeOfNonArraysToSimpleArrayRector::class,
-        ArrayKeyExistsTernaryThenValueToCoalescingRector::class,
-        AbsolutizeRequireAndIncludePathRector::class,
-        ChangeArrayPushToArrayAssignRector::class,
-        ForRepeatedCountToOwnVariableRector::class,
-        ForeachItemsAssignToEmptyArrayToAssignRector::class,
-        InlineIfToExplicitIfRector::class,
-        UnusedForeachValueToArrayKeysRector::class,
-        CommonNotEqualRector::class,
-        SetTypeToCastRector::class,
-        LogicalToBooleanRector::class,
-        VarToPublicPropertyRector::class,
-        IssetOnPropertyObjectToPropertyExistsRector::class,
-        NewStaticToNewSelfRector::class,
-        UnwrapSprintfOneArgumentRector::class,
-        SwitchNegatedTernaryRector::class,
-        SingularSwitchToIfRector::class,
-        SimplifyIfNullableReturnRector::class,
-        FuncGetArgsToVariadicParamRector::class,
-        CallUserFuncToMethodCallRector::class,
-        CallUserFuncWithArrowFunctionToInlineRector::class,
-        CountArrayToEmptyArrayComparisonRector::class,
-        FlipTypeControlToUseExclusiveTypeRector::class,
-        InlineArrayReturnAssignRector::class,
-        InlineIsAInstanceOfRector::class,
-        TernaryFalseExpressionToIfRector::class,
-        InlineConstructorDefaultToPropertyRector::class,
-        TernaryEmptyArrayArrayDimFetchToCoalesceRector::class,
-        OptionalParametersAfterRequiredRector::class,
-        SimplifyEmptyCheckOnEmptyArrayRector::class,
-        SwitchTrueToIfRector::class,
-        CleanupUnneededNullsafeOperatorRector::class,
-        DisallowedEmptyRuleFixerRector::class,
-        ConvertStaticPrivateConstantToSelfRector::class,
-        LocallyCalledStaticMethodToNonStaticRector::class,
-        NumberCompareToMaxFuncCallRector::class,
-        CompleteMissingIfElseBracketRector::class,
-        RemoveUselessIsObjectCheckRector::class,
-        StaticToSelfStaticMethodCallOnFinalClassRector::class,
-    ]);
+    // the rule order matters, as its used in withCodeQualityLevel() method
+    // place the safest rules first, follow by more complex ones
+    $rectorConfig->rules(CodeQualityLevel::RULES);
```

Comment:
We use the list of rules defined in CodeQualityLevel instead of defining the list here